src/types/ReactFlagsSelectTypes.ts

```diff
@@ -1,4 +1,6 @@
 export type Countries = Record<string, string>;
 export type CountryCodes = Array<string>;
-export type CustomLabels = Record<string, string>;
+export type CustomLabels =
+  | Record<string, string>
+  | Record<string, { primary: string; secondary: string }>;
```

Comment:
let's export a type called CustomLabel which is essentially
`{ 
    primary: string;
   secondary?: string
}`
I'll suggest secondary been optional

Reply:
Hence `customLabels = Record<string, CustomLabel | string>`